src/app/GitUI/UserControls/RevisionGrid/Graph/RevisionGraphLaneColor.cs

```diff
@@ -11,7 +11,7 @@ public static int GetColorForLane(int seed)
             return Math.Abs(seed) % PresetGraphBrushes.Count;
         }
 
-        public static Color NonRelativeColor { get; } = Color.LightGray;
+        public static Color NonRelativeColor { get; } = Color.LightGray.AdaptBackColor();
```

Comment:
🤔 Graph lines are foreground.

Reply:
Changing, but neither look good (adjustments in actual theme PR)

gray Back
![image](https://github.com/user-attachments/assets/ea001371-f65e-4dbd-8042-f8bbcee15999)

gray Fore
![image](https://github.com/user-attachments/assets/c0447ace-1e0e-48e5-8c1d-1769b058365e)

Reply:
I will make this color themable, adding to another PR

![image](https://github.com/user-attachments/assets/b30764f8-34cf-4cb4-b986-0c87c4ac3ce6)